src/request-item/request-item.tsx

```diff
@@ -0,0 +1,63 @@
+// NEED TO FIX IMG FILE LOCATION AFTER FOLDER STRUCTURE IS SET
+
+import { h } from 'preact'
+import cn from 'classnames'
+import classes from './request-item.module.css'
+// import girl from '../assets/imgs/girl.jpg'
```

Comment:
I'd generally prefer to avoid commented out code in the repository.
```suggestion
```

Also, please remove the `girl.jpg` file, as it would never be used when we actually use the library.

Reply:
Definintely. I tend to keep comments during buildout so I don't forget stuff but I always try to clean up before publishing.
In regards to the girl file, yup - I just needed something to test the css. There's no content (or app) yet and I have no idea where images will be stored so I'm improvising in the meantime. Will definitely remove.